src/Styleguide/Elements/__stories__/Typography.story.tsx

```diff
@@ -1,38 +1,35 @@
 import { storiesOf } from "@storybook/react"
 import React from "react"
-import { Theme } from "../../theme"
+import { Theme, Display, Sans, Serif } from "@artsy/palette"
+import { Col } from "../Grid"
 import { withInfo } from "@storybook/addon-info"
-import { Display, Sans, Serif } from "../Typography"
 
 const stories = storiesOf("Styleguide/Typography", module)
 
 stories
+  .addDecorator(storyFn => {
+    return <Theme>{storyFn()}</Theme>
+  })
   .add(
     "Sans",
     withInfo(`
 
     https://www.notion.so/artsy/Typography-d1f9f6731f3d47c78003d6d016c30221
 
   `)(() => {
+      // prettier-ignore
+      const sizes = ['1', '2', '3', '3t', '4', '4t', '5',
```

Comment:
Why hard code the sizes? We could just read these from the `theme`, right?

Reply:
Oh right :doh: